wurst/systems/entities/modules/Hostile.wurst

```diff
@@ -62,4 +67,10 @@ public abstract class Hostile extends UnitEntity
         else
             udg_ANIMAL_CURRENT -= 1
 
+        if this instanceof Mammoth
+            or this instanceof DiscoDuck
```

Comment:
Can you change this condition to all be on the same line? I find this confusing at a glance because it looks like `or this instanceof DiscoDuck` is part of the block of the `if` structure, rather than the condition.